src/SetupApi.Tests/SP_DRVINFO_DETAIL_DATAFacts.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using static PInvoke.SetupApi;
+using System.Runtime.InteropServices;
+using Xunit;
+
+public class SP_DRVINFO_DETAIL_DATAFacts
+{
+    [Fact]
+    public void Layout_Test()
+    {
+        if (Environment.Is64BitProcess)
```

Comment:
These values come from a unit test from our internal codebase. IIRC, they were obtained by calling `sizeof` and `offsetof` the various structs/fields in a C program.